experimenter/experimenter/nimbus_ui_new/filtersets.py

```diff
@@ -300,10 +303,14 @@ def filter_type(self, queryset, name, value):
             query |= Q(is_rollout=True)
         return queryset.filter(query)
 
-    def filter_takeaways(self, queryset, name, value):
+    def filter_takeaways(self, queryset, name, values):
         query = Q()
-        if NimbusExperiment.Takeaways.DAU_GAIN in value:
-            query |= Q(takeaways_metric_gain=True)
-        if NimbusExperiment.Takeaways.QBR_LEARNING in value:
-            query |= Q(takeaways_qbr_learning=True)
+        for value in values:
+            if value in NimbusExperiment.Takeaways:
+                if value == NimbusExperiment.Takeaways.QBR_LEARNING:
+                    query |= Q(takeaways_qbr_learning=True)
+                elif value == NimbusExperiment.Takeaways.DAU_GAIN:
+                    query |= Q(takeaways_metric_gain=True)
+            elif value in NimbusExperiment.ConclusionRecommendation:
```

Comment:
We don't need the outside if, its redundant.

```suggestion
            if value == NimbusExperiment.Takeaways.QBR_LEARNING:
                query |= Q(takeaways_qbr_learning=True)
            elif value == NimbusExperiment.Takeaways.DAU_GAIN:
                query |= Q(takeaways_metric_gain=True)
            elif value in NimbusExperiment.ConclusionRecommendation:
```

Reply:
the only reason was to group takeaways together and conclusion recommendations separately